src/scripts/object.search.js

```diff
@@ -172,12 +176,24 @@ module.exports = (robot, res) => {
 					});
 					let i = 1;
 					_.forEach(downloadedObjects, (downloadedObject) => {
+						// trainingDataMap[downloadedObject.name]
 						robot.logger.debug(`${TAG}: Temp file created for ${downloadedObject.name} at ${downloadedObject.path}`);
+						let keywordString = '';
+						_.forEach(trainingDataMap[downloadedObject.name], (word) => {
+							keywordString += word.toLowerCase() + ', ';
+						});
+						if (keywordString === '')
+							keywordString = 'None';
+						else {
+							keywordString = keywordString.substring(0, keywordString.length - 2);
+						}
 						robot.emit('ibmcloud.formatter', {
 							response: res,
 							fileName: downloadedObject.name,
 							filePath: downloadedObject.path,
-							message: `(${i}) ${downloadedObject.name} matches with a confidence of ${confidenceMap[downloadedObject.name]}%`
+							message: `${downloadedObject.name}`,
```

Comment:
Back ticking is typically beneficial when combining regular text and variables.  Since this is only variables, I'm not sure why you're back ticking.  But, of course, it still works so not a big deal.